packages/venia-ui/lib/components/Header/currencySwitcher.css

```diff
@@ -1,41 +1,102 @@
 .root {
-    composes: root from './storeSwitcher.css';
```

Comment:
A few instances in this file where we've removed the `composes` properties, and replaced with duplicated styles. Are we avoiding `composes`? Any possibility of avoiding duplication?

Reply:
The switchers are no longer identical with the introduction of store grouping. Styles that previously may have been generic/shareable are no longer. If anything, there should be some top-level `switcher.css` styles that both `storeSwitcher` and `currencySwitcher` compose from. Short of adding that scope to this PR I just duplicated the old styles to make sure nothing got lost.